resources/repos.json

```diff
@@ -1591,10 +1591,10 @@
         },
         "dependencies": {
             "Minecraft Version(s)": {
-                "1": "1.18.x, 1.19x"
+                "1": "1.17x, 1.18.x, 1.19x"
             },
             "Slimefun Version": {
-                "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#31\">RC-31 or higher</a>"
+                "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#31\">RC-32 or higher</a>"
```

Comment:
Unless this applies to build one too, these should have seperate entries.

Reply:
the minecraft version was always 1.17 compatible and i added seperate entries for the slimefun version